roles/httpd/proxy/templates/15-service.conf.j2

```diff
@@ -0,0 +1,17 @@
+
+<VirtualHost *:{{ httpd_proxy_port }}>
+
+  <Location />
```

Comment:
Since we are generalizing the proxy setting here, we could also add possibility for using prefix. Eg. by default leave location /, but in case we would want /uchiwa prefix, it should be possible to use it. Without setting prefixes (I know that kibana won't work) we won't be able to have all our UIs on single host if that use case will be required.